CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ else()
 		string(CONCAT explanation_msg
 			"The Minetest team has forked Irrlicht to make their own customizations. "
 			"It can be found here: https://github.com/minetest/irrlicht\n"
-			"For example use: git clone --depth=1 https://github.com/minetest/irrlicht lib/irrlichtmt\n")
+/			"For example use: git clone --depth=1 --branch=$(cat misc/irrlichtmt_tag.txt) https://github.com/minetest/irrlicht lib/irrlichtmt\n")
```

Comment:
It might be better to just refer to `doc/compiling/` so that the proper setup instructions are followed - and not just this specific step.

Reply:
Done.